2.10/2.10.17/index.html

```diff
@@ -14,15 +14,15 @@
 Если введенные числа правильные, тогда выводится сообщение:
 "X - Y = Z" или "X + Y = Z" взависимости от выбранной ранее операции
 <script>
-	const addOrSubtract = confirm('Что сделать? ОК - Сложить, ОТМЕНА - Отнять');
+	const isAdd = confirm('Что сделать? ОК - Сложить, ОТМЕНА - Отнять');
```

Comment:
'add' is a verb. With 'is, has, have' naming we should use a noun, process or a verb in past tense, for example:
isCompleted, isStarted, isError, isMoving, isBigWindow, isSmallText, isAddition.

this short form from questions without 'it':
Is it completed? Is it an error? Is it moving? Is it a big window? Is it small text? Is it addition?